### ISSUE TITLE
Add a timeout primitive for executing blocks after a set time interval

### ISSUE THREAD

**Issue by smarr** [smarr/SOMns#48]
Should probably look something like this in the actor module:

``` Smalltalk
(* Executes aBlock after waiting for `milliseconds`.
   Returns a promise, which is going to be resolved to the
   return value of the block. *)
public after: milliseconds do: aBlock = (
  ^ vmMirror actorsDo: aBlock after: milliseconds
)
```

**Comment by smarr** [comment]
To implement this, the simplest strategy is probably to create a promise pair in the SOM side, and pass the timeout and the resolve to the vmMirror.
The promise should get the block as a `whenResolved:`.

That way, we ensure that the block is execute on the right actor without extra complications and without having to worry that the timer is racing with the event loop.

**Comment by smarr** [comment]
Solved with #70.
